config/prod.webpack.config.js

```diff
@@ -23,10 +25,23 @@ plugins.push(
         './RootApp': resolve(__dirname, '../src/AppEntry'),
         './SystemDetail': resolve(__dirname, '../src/Modules/SystemDetail'),
       },
-    }
+    },
+    sentryWebpackPlugin({
+      authToken: process.env.SENTRY_AUTH_TOKEN,
+      org: process.env.SENTRY_ORG,
+      project: process.env.SENTRY_PROJECT,
+    })
   )
 );
 
+const SourceMapsPlugin = new webpack.SourceMapDevToolPlugin({
+  test: /\.js/i,
+  exclude: /(node_modules|bower_components)/i,
+  filename: `sourcemaps/[name].js.map`,
```

Comment:
The assets generated by the shared webpack config will use the `contenthash`. It might be important for these to match up.
https://github.com/RedHatInsights/frontend-components/blob/0e90e6071e327881def444388907a83a3d5a1b1c/packages/config/src/lib/createConfig.ts#L99

Reply:
when running everything is building out properly. I believe this is good.